integration_tests/e2e/courtCaseNextHearingDate.cy.ts

```diff
@@ -19,6 +19,10 @@ context('Next hearing date page', () => {
       .and('contain.text', 'Cell numberCELL-1')
   })
 
+  it('button to save draft is not displayed', () => {
+    courtCaseNextHearingDatePage.saveDraftButton().should('not.exist')
+  })
+
```

Comment:
was contemplating whether to test this in all the places the Save Draft button appears, but due to it's simple nature opted to just check it in one page